<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddUserDialog</class>
 <widget class="QWidget" name="AddUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加用户</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/sy.jpg</normaloff>:/sy.jpg</iconset>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>添加用户</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditId">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_class">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>班级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditClass">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAge">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSex">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_phone">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>电话号码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPhone">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_ent">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
